test/integration/analysis.js

```diff
@@ -7,6 +7,8 @@ var Analysis = require('../../lib/analysis');
 var BatchClient = require('../../lib/postgresql/batch-client');
 
 var testConfig = require('../test-config');
+var testHelper = require('../helper');
+var DataObservatoryMultipleMeasures = require('../../lib/node/nodes/data-observatory-multiple-measures');
```

Comment:
@rochoa Don't like to include specific analysis in a generic test but I didn't found a generic analysis to deal with this. Would be a good idea to create a test analysis or something like that? or leave it like this?